package.json

```diff
@@ -7,11 +7,7 @@
     "build": "npm run build-loose -- --strict",
     "build-loose": "ecmarkup --verbose spec.emu index.html"
   },
-  "homepage": "https://github.com/tc39/template-for-proposals#readme",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/tc39/template-for-proposals.git"
-  },
```

Comment:
The object form is strictly preferred, and `git+https` is strongly preferred over just https for all git operations.